pkg/cloudprovider/gce/gce.go

```diff
@@ -421,6 +464,67 @@ func (gce *GCECloud) GetNodeResources(name string) (*api.NodeResources, error) {
 	}
 }
 
+func getMetadataValue(metadata *compute.Metadata, key string) (string, bool) {
+	for _, item := range metadata.Items {
+		if item.Key == key {
+			return item.Value, true
+		}
+	}
+	return "", false
+}
+
+func (gce *GCECloud) Configure(name string, spec *api.NodeSpec) error {
+	instanceName := canonicalizeInstanceName(name)
+	instanceCall := gce.service.Instances.Get(gce.projectID, gce.zone, instanceName)
+	instance, err := instanceCall.Do()
```

Comment:
You could one-line this:
instance, err := gce.service.Instances.Get(gce.projectID, gce.zone, instanceName).Do()